app/helpers/contentful_rails/markdown_helper.rb

```diff
@@ -1,12 +1,14 @@
 module ContentfulRails
+  # Custom Redcarpert wrapper with opinionated defaults.
```

Comment:
`Redcarpet` (I think the `r` is either to much here, or you are missing the `r` in the next change...